server/paymentProviders/paypal/payment.js

```diff
@@ -123,40 +124,51 @@ export async function executePayment(order) {
 /** Create transaction in our database to reflect a PayPal charge */
 export async function createTransaction(order, paymentInfo) {
   const transaction = paymentInfo.transactions[0];
-  const amountFromPayPal = parseFloat(transaction.amount.total);
-  const paypalFee = parseFloat(get(transaction, 'related_resources.0.sale.transaction_fee.value', '0.0'));
-  const amountFromPayPalInCents = floatAmountToCents(amountFromPayPal);
-  const paypalFeeInCents = floatAmountToCents(paypalFee);
-  const currencyFromPayPal = transaction.amount.currency;
 
-  // TODO: Double check why in one case we're using amountFromPayPalInCents and in the other order.totalAmount
-  const hostFeeInHostCurrency = await getHostFee(amountFromPayPalInCents, order);
-  const platformFeeInHostCurrency = await getPlatformFee(order.totalAmount, order);
+  const amount = floatAmountToCents(parseFloat(transaction.amount.total));
+  const currency = transaction.amount.currency;
```

Comment:
The assumption here is that this currency is equal to the collective currency.
It is implicit in the API, but from the frontend, this currency is set to be `tier.amount.currency || collective.currency`, which validates the assumption.